code/modules/mob/living/living.dm

```diff
@@ -443,6 +443,12 @@
 	if (s_active && !(s_active.ClickAccessible(src, depth=STORAGE_VIEW_DEPTH) || s_active.Adjacent(src)))
 		s_active.close(src)
 
+	for(var/mob/M in oview(src))
+		if(M.client && (src in cone(M, OPPOSITE_DIR(M.dir), view(10, M))))
```

Comment:
i.e the only mob that needs to actually call update_vision cone is the actually moving mob